audioswitch/src/main/java/com/twilio/audioswitch/AudioSwitch.kt

```diff
@@ -105,21 +103,36 @@ class AudioSwitch {
      *
      * @param context An Android Context.
      * @param loggingEnabled Toggle whether logging is enabled. This argument is false by default.
-     * @param audioFocusChangeListener A listener that is invoked when the system audio focus is
-     * updated. Note that updates are only sent to the listener after [activate] has been called.
+     * @param audioFocusChangeListener A listener that is invoked when the system audio focus is updated.
+     * Note that updates are only sent to the listener after [activate] has been called.
+     * @param preferredDeviceList The order in which [AudioSwitch] automatically selects and activates
+     * an [AudioDevice]. This parameter is ignored if the [selectedAudioDevice] is not null.
+     *
+     * The default preferred [AudioDevice] order is the following:
```

Comment:
Add documentation that an `IllegalArgumentException` is thrown if the list contains duplicate entries.